src/Core/EventLogger.hpp

```diff
@@ -26,37 +26,53 @@
 #include <QFile>
 #include <QTextStream>
 
-namespace Core
-{
+#define NUMBER_OF_LOGS_TO_KEEP (50)
+#define LOG_FILE_NAME QString("cpeditor")
+#define LOG_DIR_NAME QString("cpeditorLogFiles")
+#define MAXIMUM_FUNCTION_NAME_SIZE (30)
+#define MAXIMUM_FILE_NAME_SIZE (20)
```

Comment:
You mean `constexpr variables`?

I did not did it because of adding `constexpr`. Plus any benefit of constant over define?  Form point of performance non constexpr variables are not better than define!

Reply:
I think there is no noticeable performance difference, but it's often better to use constants than defines.

- <https://stackoverflow.com/questions/4715831/why-is-define-bad-and-what-is-the-proper-substitute>
- <https://www.quora.com/Why-is-it-a-bad-practice-to-use-C-or-C++-macros>

Also, these should be put in EventLogger.cpp instead of EventLogger.hpp, because they are only used in EventLogger.cpp, and putting them in the cpp file can reduce recompiling and the possibility of name conflicting.

Reply:
Will be resolved soon.